### PR TITLE
[zombienet] more logs for smoke tests

### PR DESCRIPTION
**Pull request by drahnr**
Increase logs for smoke tests for easier verification and turnaround.

Also leaves local overrides commented, to avoid one extra roundtrip.

### CODE REVIEW

File: zombienet_tests/parachains/0001-parachains-smoke-test.toml [hunk 1]
```diff
@@ -10,10 +10,26 @@ command = "polkadot"
   name = "alice"
   extra_args = [ "--alice" ]
 
+  [[relaychain.nodes.env]]
+  name = "RUST_LOG"
+  value = "parachain=debug"
+
+#  [[relaychain.nodes.overrides]]
+#  local_path = "../../target/release/polkadot"
+#  remote_name = "wonderland"
```
Comment (pepoviola) [inline zombienet_tests/parachains/0001-parachains-smoke-test.toml:19]: Hi @drahnr, just a comment on this config. `Zombienet` use remote_name to create the `file` in the container, in this case `wonderland` **but** this config **use** the default command (e.g `polkadot`) `so` wonderland will not be used.

Related issue in `zombienet` https://github.com/paritytech/zombienet/issues/46 to explore make this optional.
Thx!

**Review by pepoviola — APPROVED** [review]

File: zombienet_tests/parachains/0001-parachains-smoke-test.toml (hunk 1 above)
Comment (drahnr) [reply zombienet_tests/parachains/0001-parachains-smoke-test.toml:19]: In that case, `zombienet` should point me to precisely this conflict with line numbers :upside_down_face:

Comment (pepoviola) [reply zombienet_tests/parachains/0001-parachains-smoke-test.toml:19]: Sounds good! I open this [issue](https://github.com/paritytech/zombienet/issues/57) to track this.
Thanks!

**Comment by drahnr** [comment]
@sandreim please merge unless this is obsoleted by your work, in that case please close

**Comment by sandreim** [comment]
closed  since https://github.com/paritytech/polkadot/pull/4901

**Pull request closed without merging**